buildSrc/src/main/groovy/org/elasticsearch/gradle/test/AntFixture.groovy

```diff
@@ -149,11 +149,10 @@ public class AntFixture extends AntTask implements Fixture {
         }
 
         // the process is started (has a pid) and is bound to a network interface
-        // so now wait undil the waitCondition has been met
-        // TODO: change this to a loop?
+        // so now evaluates if the waitCondition is successful
         boolean success
         try {
-            success = waitCondition(this, ant) == false
+            success = waitCondition(this, ant)
```

Comment:
lol, good catch. Can you create a separate PR for this?

Reply:
Sure, I'll do that.

Reply:
I created https://github.com/elastic/elasticsearch/pull/31272